bin/license.py

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/env python3
```

Comment:
Hi @zhongjiajie, thanks for your comment, however this script are using Python 3, and Python 2 is no longer under maintenance status, so IMO specific version of Python is still necessary

Reply:
Do not get me wrong. I do not mean we should support version 2
It is not related to version 2 or version 3, what if user with `python3` in env, just `python3.6` or `python3.10` but with a soft link `python -> python3.6`?
I think we should use the exactly word `python`, instead of pointing `python3`

and then check the version in laster

Reply:
@zhongjiajie Oh, I see. In that case, SGTM.